### PR TITLE
Decrease priority of bisect jobs in accordance with investigation jobs

### PR DESCRIPTION
**Pull request by Martchus**
See https://progress.opensuse.org/issues/152281

### CODE REVIEW

**Review by perlpunk — APPROVED** [review]

File: openqa-trigger-bisect-jobs [hunk 1]
```diff
@@ -121,6 +126,19 @@ def openqa_comment(job, host, comment, dry_run):
     ]
     return call(args, dry_run)
 
+def openqa_set_job_prio(job_id, host, prio, dry_run):
```
Comment (mimi1vx) [inline openqa-trigger-bisect-jobs:129]: I don't think this whole part is needed

**Comment by okurz** [comment]
> It creates new openqa api call ?
> 
> prio shuld be set during clone call by setting `_PRIORITY=<needed_prio>` without need second api call to set different priority

that works? Well, then that sounds good :)

File: openqa-trigger-bisect-jobs [hunk 2]
```diff
@@ -292,6 +311,7 @@ def main(args):
         for job_id in sorted(created_job_ids):
             log.info(f"Created {job_id}")
             created += f"* **{test_name}**: {base_url}/t{job_id}\n"
+            openqa_set_job_prio(job_id, args.url, prio, args.dry_run)
```
Comment (mimi1vx) [inline openqa-trigger-bisect-jobs:314]: really this isn't needed

**Review by mimi1vx — CHANGES_REQUESTED** [review]

File: openqa-trigger-bisect-jobs (hunk 2 above)
Comment (Martchus) [reply openqa-trigger-bisect-jobs:314]: Is it really possible to set the priority with `openqa-clone-job`? It doesn't seem to be possible according to `--help`.

**Comment by perlpunk** [comment]
I just tried
```
openqa-clone-job --host http://localhost:9526 --from https://openqa.opensuse.org/tests/3792472 _PRIORITY=124
```
and the prio of the created job was 50...

**Comment by Martchus** [comment]
I don't think you're doing anything wrong. To my knowledge this is simply not implemented in `openqa-clone-job` (or the underlying API route). Note that this would have worked as scheduling parameter but that's a whole different route.

**Comment by mimi1vx** [comment]
ah, @Martchus  is right ... it works in normal scheduling but not for clone-job :( Which is pretty inconsistent..

other parameters like `_GROUP` works

**Pull request merged**